cypress/integration/collections_spec.js

```diff
@@ -0,0 +1,123 @@
+const ace = require('brace');
+require('brace/mode/javascript');
+require('brace/theme/monokai');
+
+
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Dashboard Collections Page', () => {
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit('/#/collections');
+      shouldBeRedirectedToLogin();
+    });
+  });
+
+  describe('When logged in', () => {
+    const name = 'TESTCOLLECTION';
+    const version = '006';
+    beforeEach(() => {
+      cy.login();
```

Comment:
Ideally, I think we should also `cy.task('resetState')` here, as it means the tests are really well isolated and not dependent on each other's state

Reply:
I have used the `TESTCOLLECTION` through out the test suite.